prod-deployment.md

```diff
@@ -94,124 +118,150 @@ The default username for the jump host is `azureadmin` and the password was set
+    az login
+    ```
+    
+    ```pwsh
+    $AZURE_SUBSCRIPTION_ID = ((azd env get-values --output json | ConvertFrom-Json).AZURE_SUBSCRIPTION_ID)
+    ```
 
-From the jump host, launch Windows Terminal to setup required tools:
+    ```pwsh
+    az account set --subscription $AZURE_SUBSCRIPTION_ID
+    ```
 
-1. Install the [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd):
 
+1. Run the following to set the environment variables for the bastion tunnel:
+
+    ```pwsh
+    $bastionName = ((azd env get-values --output json | ConvertFrom-Json).BASTION_NAME)
+    $resourceGroupName = ((azd env get-values --output json | ConvertFrom-Json).BASTION_RESOURCE_GROUP)
+    $targetResourceId = ((azd env get-values --output json | ConvertFrom-Json).JUMPHOST_RESOURCE_ID)
+    ```
+
+1. Run the following command to create a bastion tunnel to the jump host:
+    ```pwsh
+    az network bastion tunnel --name $bastionName --resource-group $resourceGroupName --target-resource-id $targetResourceId --resource-port 22 --port 50022
+    ```
+    
+    > **NOTE**
+    >
+    > Now that the tunnel is open, change back to use the original PowerShell session.
+
+
+1. From PowerShell use the following SCP command to upload the code to the jump host (use the password you retrieved from Key Vault to authenticate the SCP command):
     ```shell
-    powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"
+    scp -r -P 50022 * azureadmin@127.0.0.1:web-app-pattern/
     ```
 
-1. Install the [dotnet SDK](https://learn.microsoft.com/dotnet/core/tools/dotnet-install-script):
+    > [Remote host identification has changed](troubleshooting.md#remote-host-identification-has-changed)
 
+1. From PowerShell use the SCP command to upload the AZD environment to the jump host:
     ```shell
-    powershell -ex AllSigned -c "Invoke-RestMethod 'https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.ps1' -OutFile dotnet-install.ps1"
+    scp -r -P 50022 ./.azure azureadmin@127.0.0.1:web-app-pattern/
     ```
 
+1. Run the following command to start a shell session on the jump host:
+
     ```shell
-    .\dotnet-install.ps1 -Version 8.0.101
+    ssh azureadmin@127.0.0.1 -p 50022
     ```
 
-1. Add dotnet to the path environment variable
+### 4. Deploy code from the jump host
 
-    ![#Add dotnet to the path variable](./assets/images/jumphost-path-setup.png)
+1. Change to the directory where you uploaded the code:
 
-    Add the path: `%USERPROFILE%\AppData\Local\Microsoft\dotnet`.
+    ```shell
+    cd web-app-pattern
+    ```
 
-1. Add Nuget source
+1. Change the exeuatable permissions on the scripts:
+    <!-- set script permissions is required when running from windows. not required from Dev Container experience -->
 
     ```shell
-    dotnet nuget add source https://api.nuget.org/v3/index.json -n nuget.org
+    chmod +x ./infra/scripts/predeploy/call-set-app-configuration.sh
```

Comment:
`chmod +x ./infra/scripts/**/*.sh instead`?  Then we only need one call and won't have to update the docs when we add the ACA secret update script.